lib/coinbase/address/developer_address.rb

```diff
@@ -0,0 +1,189 @@
+# frozen_string_literal: true
+
+require 'bigdecimal'
+require 'eth'
+require 'jimson'
+
+module Coinbase
+  # A representation of a blockchain Address, which is a user-controlled account on a Network. Addresses are used to
+  # send and receive Assets, and should be created using Wallet#create_address. Addresses require an
+  # Eth::Key to sign transaction data.
+  class DeveloperAddress < Address
```

Comment:
Should this be DeveloperAddress or WalletAddress?

Reply:
+1, I think WalletAddress is probably the right terminology here.

A `WalletAddress` that belongs to a managed wallet, and could be attributed to an end-user on the developer's side.